doc/html_publisher.rst

```diff
@@ -0,0 +1,44 @@
+Integrate collapsible log into Jenkins
+======================================
+
+The `--html-log <args.html#Output>`__ option enables generating an HTML page with collapsible blocks,
+very convenient when navigating a long build log of a `Universum` run with many steps.
+
+While TeamCity has its own mechanisms increasing readability, and Universum ran locally by default just stores
+step execution logs to files to be accessed separately, Jenkins just provides the full build log, formatted as
+a very long text with no real anchors. Generating a user-friendly log and integrating it into a `Universum` job
+simplifies interaction with `Univesrum` in CI process.
+
+One of possible ways to integrate an existing log to Jenkins job is `htmlpublisher
```

Comment:
Just omit the 'One of possible ways'
To integrate an existing log to Jenkins job use blabla. After installing it, apply blabla.